sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -42,37 +37,21 @@ class DatalabAppElement extends Polymer.Element {
   public page: string;
 
   /**
-   * Pattern for extracting current pathname component. This is matched
-   * against current location to extract the page name.
-   */
-  public rootPattern: string;
-
-  /**
-   * Current matching result from the window.location against the
-   * root pattern. This gets re-evaluated every time the current page
-   * changes, and can be used to get the current active page's name.
+   * Current matching result from the window.location. This gets re-evaluated
+   * every time the current page changes, and can be used to get the current
+   * active page's name.
    */
   public routeData: object;
 
+  // TODO: doc
```

Comment:
I don't think this should be left as a TODO

Reply:
Weird! I was sure I fixed this before sending, looks like I accidentally reverted that.